relentless/simulate/lammps.py

```diff
@@ -820,8 +820,8 @@ def to_commands(self, sim):
                  ' v_ensemble_Lx v_ensemble_Ly v_ensemble_Lz'
                  ' v_ensemble_xy v_ensemble_xz v_ensemble_yz'
                  ' mode scalar ave running'
-                 ' file {filename} overwrite format "%.16e"').format(every=self.check_thermo_every,
-                                                                     filename=sim[self].thermo_file)
+                 ' file {filename} overwrite format "% .16e "').format(every=self.check_thermo_every,
```

Comment:
This looks funny, I think it should be like this? (To prepend the white space)
```suggestion
                 ' file {filename} overwrite format " %.16e"').format(every=self.check_thermo_every,
```